app/api/event_tag_endpoint.py

```diff
@@ -50,3 +52,13 @@ async def update_tags(event_type: str):
     return {
         "total": update_result["updated"]
     }
+
+
+@router.delete("/event/tag/delete/{event_type}", tags=["event"],
+               include_in_schema=server.expose_gui_api, response_model=dict)
+async def delete_record(event_type: str):
+    try:
+        delete_result = await storage.driver.tag.delete(event_type)
+    except StorageException as e:
+        raise HTTPException(status_code=500, detail=str(e))
+    return {"deleted": 0 if delete_result is None else 1}
```

Comment:
See if driver does not return how many records were deleted. Do not hard-code it.